.github/workflows/package_installers.yml

```diff
@@ -25,6 +25,7 @@ jobs:
     name: Build and upload packaged app
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
Added this to make sure at least _some_ installers are uploaded in case of a os-specific problem with the packaging.

Reply:
Have you thought of putting this as a comment in the file?

Reply:
Done!